### DISCUSSION TITLE
web display

### DISCUSSION THREAD

**Discussion by BelaV2** [cc-tweaked/CC-Tweaked#1105] [Q&A]
i cant do web display 
hereis the link https://metsaluikandre.wixsite.com/website

**Comment by BelaV2** [comment]
i cant do monitor display whit text please help

**Reply by samuelh2005** [reply]
There has never been anything like a web display in Computer Craft. Try the Web Displays mod, not sure if it exists in modern Minecraft versions.

If you still want to use CC then you must parse and display HTML / CSS with Lua. This would be a complicated process and hasn't been done very well before.
